proc-macro/src/lib.rs

```diff
@@ -30,6 +30,54 @@ use synstructure::{
     Structure,
 };
 
+/// Register type sizes for [EventsDecoder](struct.EventsDecoder.html)
+///
+/// The `module` macro registers the type sizes of the associated types of a trait so that [EventsDecoder](struct.EventsDecoder.html) can decode events of that type when received from Substrate.
+///
+/// If you do not want an associated type to be registered, likely because you never expect it as part of a response payload to be decoded, use `#[module(ignore)]` on the type.
+///
+/// Example:
+///
+/// ```ignore
+/// #[module]
+/// pub trait Herd: Husbandry {
+/// 	type Hooves: HoofCounter;
+/// 	type Wool: WoollyAnimal;
+/// 	#[module(ignore)]
+/// 	type Digestion: EnergyProducer + std::fmt::Debug;
```

Comment:
Indentation

Reply:
Sorry, I don't see it, what is not right here?

Reply:
Looks like 1 space instead of 4
![image](https://user-images.githubusercontent.com/75586/86730254-3a711080-c026-11ea-8495-cca5c5830b52.png)

Reply:
Oh, we use spaces here, good catch. Odd that my github shows me the right white space (as did my editor, but you're right, I had one space and one tab here).